misk/web/tabs/web-actions/src/containers/WebActionsContainer.tsx

```diff
@@ -0,0 +1,237 @@
+import {
+  Card,
+  H3,
+  H5,
+  Icon,
+  Intent,
+  Menu,
+  MenuItem,
+  Tag,
+  Tooltip
+} from "@blueprintjs/core"
+import { IconNames } from "@blueprintjs/icons"
+import { FlexContainer } from "@misk/core"
+import { simpleSelect, simpleType } from "@misk/simpleredux"
+import * as React from "react"
+import { connect } from "react-redux"
+import styled from "styled-components"
+import { IDispatchProps, IState, rootDispatcher, rootSelectors } from "../ducks"
+
+export interface IWebAction {
+  applicationInterceptors: string[]
+  dispatchMechanism: string
+  function: string
+  functionAnnotations: string[]
+  name: string
+  networkInterceptors: string[]
+  parameterTypes: string[]
+  pathPattern: string
+  requestMediaTypes: string[]
+  responseMediaType: string
+  returnType: string
+}
+
+const HttpMethodTag = styled.span`
+  float: right;
+`
+
+const Column = styled.div`
+  flex-grow: 1;
+  flex-basis: 0;
+  min-width: 320px;
+`
+
+const MenuItemProps = {
+  multiline: true
+}
+
+const ComingSoon = () => (
+  <span>
+    <span>{"Coming soon "}</span>
+    <Icon icon={IconNames.AIRPLANE} />
+  </span>
+)
+
+const RequestResponeContentTypes = (props: { action: IWebAction }) => (
+  <span>
+    <span>{props.action.requestMediaTypes}</span>{" "}
+    <Icon icon={IconNames.ARROW_RIGHT} />{" "}
+    <span>{props.action.responseMediaType}</span>
+  </span>
+)
+
+enum HttpMethod {
```

Comment:
seems a bit strange that we have to re-define http methods. Is this nowhere else?

Reply:
Agreed, it should be upstreamed to `@misk/core`, couldn't find it at first look in any of our dependency libraries that we could use out of the box.